libs/model/src/models/quest.ts

```diff
@@ -0,0 +1,101 @@
+import * as schemas from '@hicommonwealth/schemas';
+import Sequelize from 'sequelize';
+import { z } from 'zod';
+import type { ModelInstance } from './types';
+
+export type QuestAttributes = z.infer<typeof schemas.Quest>;
+export type QuestInstance = ModelInstance<QuestAttributes>;
+
+export const Quest = (
+  sequelize: Sequelize.Sequelize,
+): Sequelize.ModelStatic<QuestInstance> =>
+  sequelize.define<QuestInstance>(
+    'Quest',
+    {
+      id: { type: Sequelize.INTEGER, autoIncrement: true, primaryKey: true },
+      community_id: { type: Sequelize.STRING, allowNull: false },
+      name: { type: Sequelize.STRING, allowNull: false },
```

Comment:
Shouldn't the name and description be on the questActionMeta level? Like the same questActionMeta can have different names and descriptions between communities?

Reply:
Action metadata is to configure the quest with the set of possible actions from a fixed list of events and settings, the quest should have its own name and desc.

We should be able to map event names to descriptions for Actions in the UI or schemas, but still TBD